python_modules/libraries/dagster-fivetran/dagster_fivetran/v2/__init__.py

```diff
@@ -1 +1,2 @@
 from dagster_fivetran.v2.resources import FivetranWorkspace as FivetranWorkspace
+from dagster_fivetran.v2.translator import DagsterFivetranTranslator as DagsterFivetranTranslator
```

Comment:
Until the previous functions and resource are deprecated and removed, users would import the reworked integration with the following:

`from dagster_fivetran.v2 import DagsterFivetranTranslator, FivetranWorkspace`

The new version of the integration won't actually be v2, so perhaps another import path would be better? 

This is mainly to avoid name collisions between both previous and new translators. We could also name this translator differently if we really want users to import with `from dagster_fivetran import FivetranWorkspace`, but `Dagster[IntegrationName]Translator` is the naming convention for new integrations.

Reply:
I think we should just call it something different than `*.v2`. Maybe `experimental` for a release -> 1.0 the integration, then we switch the old stuff to be under `.legacy` and make the new stuff exist under the top level package. @PedramNavid and @C00ldudeNoonan would probably have thoughts.

Reply:
That makes sense - I updated the folder name for `experimental` in the previous PR in this stack.